pinot-core/src/main/java/org/apache/pinot/core/util/TableConfigUtils.java

```diff
@@ -371,6 +371,7 @@ private static void validateIndexingConfig(@Nullable IndexingConfig indexingConf
     }
     Map<String, String> columnNameToConfigMap = new HashMap<>();
     Set<String> noDictionaryColumnsSet = new HashSet<>();
+    String STAR_TREE_CONFIG_NAME = "StarTreeIndex Config";
```

Comment:
make this static?

Reply:
I thought about it - but this static variable will only be used within this method and nowhere else. So decided to let it be.